addons/web_editor/models/ir_ui_view.py

```diff
@@ -84,9 +84,10 @@ def save_oe_structure(self, el):
             'arch': self._pretty_arch(arch),
             'key': '%s_%s' % (self.key, el.get('id')),
             'type': 'qweb',
+            'mode': 'extension',
         }
         vals.update(self._save_oe_structure_hook())
-        self.create(vals)
+        self.env['ir.ui.view'].create(vals)
```

Comment:
I'm curious about this change 🤔 

Is there any real difference on doing it either way?

Reply:
Yes, it could have been enough to fix the issue to only change that. Because of the `_compute_defaults` misimplementation, see the commit message :wink:

Reply:
And in any case, if we ever fix that method, it will be in master ;)

Reply:
So do you mean that the `values` dict passed to `_compute_defaults` in this line is different depending on whether `create()` is called from an empty recordset or a non-empty one?

https://github.com/odoo/odoo/blob/3a9d1f9dfcbaeef77ae70d7c8cdc9b31dd0b7470/odoo/addons/base/models/ir_ui_view.py#L425

😵 that's weird.